pkg/util/initsystem/initsystem.go

```diff
@@ -42,8 +45,63 @@ type InitSystem interface {
 	ServiceIsActive(service string) bool
 }
 
+type OpenRCInitSystem struct{}
+
+func (openrc OpenRCInitSystem) ServiceStart(service string) error {
+	args := []string{service, "start"}
+	return exec.Command("rc-service", args...).Run()
+}
+
+func (openrc OpenRCInitSystem) ServiceStop(service string) error {
+	args := []string{service, "stop"}
+	return exec.Command("rc-service", args...).Run()
+}
+
+func (openrc OpenRCInitSystem) ServiceRestart(service string) error {
+	args := []string{service, "restart"}
+	return exec.Command("rc-service", args...).Run()
+}
+
+func (openrc OpenRCInitSystem) ServiceExists(service string) bool {
+	args := []string{service, "status"}
+	outBytes, _ := exec.Command("rc-service", args...).CombinedOutput()
```

Comment:
I can add the comment. To me it seemed weird that systemd does not output error to stderr.
Both init systems are weird here.
Error should be printed to stderr, and the exist code should no be 0.
Also, if both did behave there would be no need to parse the text. It would be enough the check the exist status of the process.

Reply:
Actually, openrc does not return 0 if a service is not found. It returns -1 if the service is not found.
In contrast to systemd, it prints the error to the standard error, and this is why I check the combinedout put.

Reply:
If the service exists but it is not enabled , the exist code is 3.

Reply:
I added the commit the latest commit.